xclim/indices/_threshold.py

```diff
@@ -232,6 +235,56 @@ def cold_spell_frequency(
     return out
 
 
+@declare_units(tas="[temperature]", thresh="[temperature]")
+def cold_spell_max_length(
+    tas: xarray.DataArray,
+    thresh: Quantified = "-10 degC",
+    window: int = 5,
+    freq: str = "AS-JUL",
+    op: str = "<",
+    resample_before_rl: bool = True,
+) -> xarray.DataArray:
+    r"""Longest cold spell.
+
+    Longest spell of temperatures over a given period.
```

Comment:
```suggestion
    Longest spell of low temperatures over a given period.
```

Reply:
I changed the docstring similar to `hot_spell_max_length`.